lua/Events.lua

```diff
@@ -83,4 +92,13 @@ function library.addEventHandlers(obj)
 
   -- Returns true to indicate success
   return true
+end
+
+-- Removes all events from registered event handlers
+library.clearEventsHandlers = function()

```

Comment:
Nitpick too, but would use `function library.clearAllEventHandlers()` instead, just for consistency and also previous point...